### PR TITLE
[Sub Rogue] Denial of the Half Giants

### PR DESCRIPTION
**Pull request by TsaboTavok**
Adding Denial of the Half Giants.
Also refactoring the on_spend() method to be a fabricated event as advised by Serf.

### CODE REVIEW

File: src/Parser/Rogue/Subtlety/Modules/Items/T21_2P.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    combatants: Combatants,
+		spellUsable: SpellUsable,
+  };
+
+  on_byPlayer_spendresource(event){
+    const spent = event.resourceChange;
+    if(event.resourceChangeType !== ResourceTypes.COMBO_POINTS) return;
+
+    if(this.combatants.selected.hasBuff(SPELLS.SUB_ROGUE_T21_2SET_BONUS.id 
+      && this.spellUsable.isOnCooldown(SPELLS.SYMBOLS_OF_DEATH.id))){
```
Comment (Pewtro) [inline src/Parser/Rogue/Subtlety/Modules/Items/T21_2P.js]: Use on_initialized() { } to check for tier bonus' so it doesn't show if the person doesn't have it equipped

Comment (TsaboTavok) [reply src/Parser/Rogue/Subtlety/Modules/Items/T21_2P.js]: Nothing is showing up for this atm anyway, but  changed it for future.

**Comment by TsaboTavok** [comment]
Soo, how about merging this?

**Comment by kfinch** [comment]
Pewtro can't because Travis is messed up (build must pass on PR to be merged). I can due to admin privilege, but I'm going to wait for the issues on master to be resolved, just to be sure. Sorry for delay.

**Pull request merged**